ibm_catalog.json

```diff
@@ -63,6 +63,174 @@
                 "key": "existing_cos_kms_key_crn",
                 "required": true
               },
+              {
+                "key": "log_analysis_plan",
+                "required": true,
+                "default_value": "7-day",
```

Comment:
you can remove `default_value` here so we don't have to maintain it in 2 places. If not specified here, it will pick the default from terraform.